build-local.rb

```diff
@@ -55,10 +63,21 @@ def build_image()
         2>&1"
     output = `#{cmd}`
     if $?.success?
-      @spinner.success
+      if @write_success_log
+        @spinner.success(" - see #{@script_dir}/logs/#{@repo_name}_build.log")
+        @putput = @output + output + "\nBuild failed... Unable to retry.\n" 
```

Comment:
Is this a typo `@putput` instead of `@output`

Reply:
Fixed